docs/docs-site/content/user/concept/_index.md

```diff
@@ -66,6 +66,14 @@ Shared documents will show-up with a little blue icon.
 
 ![Sharing](https://cdn.gethue.com/uploads/2019/04/editor_sharing.png)
 
+Along with document sharing, there is another query sharing option via links which is quicker and easier - ![Public Links/Gist Sharing](https://gethue.com/blog/2020-03-04-datawarehouse-database-sql-collaboration-and-sharing-with-link-and-gist/)
+
+### Share to Slack
+Currently in **Beta**
+
+This integration increases the collaboration with others via Slack. It expands the ability to share query links or gists to the desired slack channels which then unfurls in a rich preview for other members to view.
```

Comment:
slack --> Slack

Reply:
Currently just gave a big picture for the beta, this can be further explained after v1 @romainr

Reply:
I know, my main comment is below for the other section